jiant/tasks/seq2seq.py

```diff
@@ -22,14 +23,18 @@
 
 
 @register_task("seg_wix", rel_path="seg/wix/", max_targ_v_size=200)
+@register_task("wmt14_en_de", rel_path="wmt14/en_de/", max_targ_v_size=40000)
 class Seq2SeqTask(SequenceGenerationTask):
     """Sequence-to-sequence Task"""
 
     def __init__(self, path, max_seq_len, max_targ_v_size, name, **kw):
         super().__init__(name, **kw)
-        self.scorer2 = BooleanAccuracy()
+        if name == "seg_wix":
```

Comment:
Cleaner to make this a keyword argument (like max_targ_v_size) rather than a switch on name.

Reply:
Done.